yarn-project/pxe/src/contract_function_simulator/proxied_node.ts

```diff
@@ -7,8 +8,12 @@ import type { NodeStats } from '@aztec/stdlib/tx';
  */
 export type ProxiedNode = AztecNode & { getStats(): NodeStats };
 
+export type NodeOverrides = {
```

Comment:
What is the purpose of this?

Reply:
This was ultimately not needed, removing